src/components/Icon.tsx

```diff
@@ -703,6 +703,16 @@ export const Icons = {
       <path d="M 5.90625 10.800781 L 18.15625 10.800781 C 18.410156 10.800781 18.632812 10.578125 18.632812 10.328125 C 18.632812 10.074219 18.410156 9.851562 18.15625 9.851562 L 5.90625 9.851562 C 4.925781 9.851562 4.136719 9.0625 4.136719 8.085938 C 4.136719 7.105469 4.925781 6.316406 5.90625 6.316406 C 6.882812 6.316406 7.671875 7.105469 7.671875 8.085938 C 7.671875 8.335938 7.894531 8.558594 8.148438 8.558594 C 8.398438 8.558594 8.621094 8.335938 8.621094 8.085938 C 8.621094 6.601562 7.390625 5.367188 5.90625 5.367188 C 4.421875 5.367188 3.191406 6.601562 3.191406 8.085938 C 3.191406 9.570312 4.421875 10.800781 5.90625 10.800781 Z M 5.90625 10.800781 " />
     </>
   ),
+  circle: (
+    <>
+      <path d="M12 2C6.486 2 2 6.486 2 12C2 17.514 6.486 22 12 22C17.514 22 22 17.514 22 12C22 6.486 17.514 2 12 2Z" />
+    </>
+  ),
+  checkCircleFilled: (
+    <>
+      <path d="M12 2C6.486 2 2 6.486 2 12C2 17.514 6.486 22 12 22C17.514 22 22 17.514 22 12C22 6.486 17.514 2 12 2ZM10.001 16.413L6.288 12.708L7.7 11.292L9.999 13.587L15.293 8.293L16.707 9.707L10.001 16.413Z" />
+    </>
```

Comment:
Can we remove the unnecessary `<>` fragments from around these path properties, please?